Networking/Sources/Networking/Services/Emoji/EmojiService.swift

```diff
@@ -9,14 +9,14 @@ import Foundation
 
 public protocol EmojiServiceable {
     func getEmoji( _ name: String,
-                  _ length: String) async throws -> [Emoji]
+                  _ length: String) async throws -> Emoji
```

Comment:
that's some wacky multiline formatting
btw, check out https://github.com/aheze/Multiliner, this extension can really help with easily multilining anything